### ISSUE TITLE
different tensor dimensions from original paper in generator

### ISSUE THREAD

**Issue by andreistirb** [carpedm20/DCGAN-tensorflow#273]
Hello, I noticed that the generator and the sampler have different dimensions than in the original paper. 
For example, in sampler, `h0 = tf.reshape(linear(z, self.gf_dim*8*s_h16*s_w16, 'g_h0_lin'), [-1, s_h16, s_w16, self.gf_dim * 8])`, I think self.gf_dim should be multiplied by 16 in order to reproduce the dimensions in the original paper. Is there any reason behind multiplying it only by 8 and thus having only half of that dimension?

Thank you very much!

**Comment by MihaiDogariu** [comment]
@andreistirb You are right, the highest depth dimension that the networks reach, excluding the flattening layer, is 512. I don't know if it brings a great improvement. Have you tried adding another layer?

**Comment by andreistirb** [comment]
@MihaiDogariu I haven't tried adding another layer, I just modified the dimensions of the current ones. The  default values of df_dim and gf_dim in the constructor of DCGAN class are 64 and I tried with 128 in order to have the same dimensions as in the original paprer.

**Comment by MihaiDogariu** [comment]
@andreistirb And did that produce any significant change in the result?

**Comment by andreistirb** [comment]
@MihaiDogariu I can not respond to your question because I do not know how to quantify the changes. :D
